cmake/partition_manager.cmake

```diff
@@ -158,9 +158,13 @@ if (CONFIG_PM_EXTERNAL_FLASH)
     )
 endif()
 
-set(pm_out_partition_files ${ZEPHYR_BINARY_DIR}/../partitions_${DOMAIN}.yml)
-set(pm_out_region_files ${ZEPHYR_BINARY_DIR}/../regions_${DOMAIN}.yml)
-set(pm_out_dotconf_files ${ZEPHYR_BINARY_DIR}/../pm_${DOMAIN}.config)
+if (DOMAIN)
+  set(domain_suffix _${DOMAIN})
```

Comment:
just as `<var_name>_UPPER` is encouraged, for same `<var_name>` variables but in upper casing, then maybe `UNDERSCORE_DOMAIN` would be better here, just to indicate origin of the variable.

```suggestion
  set(UNDERSCORE_DOMAIN _${DOMAIN})
```